e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -37,7 +37,7 @@ describe("scenarios > collection defaults", () => {
 
       cy.visit("/");
 
-      cy.viewport(800, 400);
+      cy.viewport(800, 500);
```

Comment:
@alxnddr I haven't seen this on time.
If you have to resize the screen, please do not do it after you visit the page. You can set the viewport in the test config and avoid re-rendering (or at least set it **before** you visit the page).

Reply:
See this example:
https://github.com/metabase/metabase/pull/28572/files#diff-e6ce19b51ea03567e3bc62bc1bec39b176bdd4324d35e959a281b7f58d15da9aR90